packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -34,12 +35,21 @@ const prepareHeaders = async (headers: Headers) => {
 export const orchestratorApi = createApi({
     reducerPath: 'orchestratorApi',
     baseQuery: (args, api, extraOptions: ExtraOptions) => {
-        const { baseQueryType } = extraOptions || {};
+        const { baseQueryType, customBaseQueryType } = extraOptions || {};
 
         const state = api.getState() as RootState;
         const { orchestratorApiBaseUrl, graphqlEndpointCore } =
             state.orchestratorConfig;
 
+        const customBaseQueries = state.customBaseQueries;
+
+        if (customBaseQueries) {
```

Comment:
This can be simplified, basicly you are performing a `find` here in an array that could also be undefined